include/eve/module/core/regular/impl/is_less.hpp

```diff
@@ -0,0 +1,42 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+namespace eve::detail
+{
+  template<typename T, typename U, callable_options O>
+  EVE_FORCEINLINE constexpr common_logical_t<T, U> is_less_(EVE_REQUIRES(cpu_), O const& o, T a, U b) noexcept
+  {
+    if constexpr (O::contains(definitely))
```

Comment:
This seems like handling of integers won't be correct.
I'm not sure what this option does but for integrals, we should probably just drop it and restart.